app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -134,7 +164,7 @@ To start your application for student loans, you need to have chosen your teache
 
 * tuition fee loans
 * maintenance loans to help with living costs
-* extra help if you have a disability, children or an adult dependant
+* extra help if you have a disability, children or an adult dependent
```

Comment:
I think `dependant` is correct - ie no change is needed?

Reply:
Yep it's dependant so no change needed